src/test/java/tsunami/security/scanner/utilities/ApplicationArgsTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tsunami.security.scanner.utilities;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth8.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ApplicationArgsTest {
+
+  @Test
+  public void testApplicationArgs() {
+    String[] args = setArgs();
+
+    ApplicationArgs jArgs = new ApplicationArgs();
+    JCommander cmd = JCommander.newBuilder().addObject(jArgs).build();
+    cmd.parse(args);
+
+    String appName = jArgs.getName();
+    String appConfigPath = jArgs.getConfigPath();
+    String templateData = jArgs.getTemplateData();
+
+    assertThat(appName).isEqualTo(args[1]);
```

Comment:
Instead of comparing with `args[n]`, please make it explicit like `assertThat(jArgs.getName()).isEqualTo("jupyter")`